multiversx_sdk_cli/cli_transactions.py

```diff
@@ -65,6 +67,7 @@ def create_transaction(args: Any):
     cli_shared.check_guardian_and_options_args(args)
     cli_shared.check_broadcast_args(args)
     cli_shared.prepare_nonce_in_args(args)
+    cli_shared.prepare_chain_id_in_args(args)
```

Comment:
I think we have to check all occurrences of `prepare_nonce_in_args` etc. and accompany them with the new `prepare_chain_id_in_args`?